### DISCUSSION TITLE
Reduce screen brightness to (close to) zero?

### DISCUSSION THREAD

**Discussion by steve8x8** [OpenNeptune3D/OpenNept4une#92] [Q&A]
I may have lost track of the recent changes, but I'm missing a means to dim the screen to an even bigger amount than the stock firmware ever allowed. Is this possible at all?
Is it feasible to "wake" the screen on touch, i.e., switch the light off (almost) completely - deactivating any "action" touches and bring the screen back to life if it's touched, perhaps with some extra "unlock" button/swipe (I'm imagining something similar to the screen-off feature of TWRP Android recovery where you have to swipe in a defined region, to unlock the screen after a timeout)?

**Comment by phillipthelen** [comment]
The screens do offer a sleep mode. They have no swipe gestures/detection, so that wouldn't be possible. But implementing something where the display is "off" until tapped might be possible.
